Include parallelize :threads option in template when fork not available (#38483)
rails/rails: guides/source/testing.md

end
```

> Rails applications generated from Ruby implementations that do not support fork will automatically include the `with: :threads` option.

Reviewer: I think we should give concrete examples here, even if the actual check no longer sniffs the platform - most readers probably have no idea whether their Ruby implementation supports fork or not.

Let's go back to what you had before:

```suggestion
Rails applications generated from JRuby or TruffleRuby will automatically include the `with: :threads` option.
 ```

Author: @eugeneius I've updated back to the concrete example.
